x-pack/test/functional/services/ml/api.ts

```diff
@@ -212,6 +212,14 @@ export function MachineLearningAPIProvider({ getService }: FtrProviderContext) {
       return body.hits.hits.length > 0;
     },
 
+    async addForecast(jobId: string, requestBody: { duration: string }): Promise<void> {
+      const { body, status } = await kbnSupertest
```

Comment:
nit: Would be nice to have some debug statements indicating we are creating a forecast similar to in `createIndex`. For example:
 ```
log.debug(`Creating forecast for ${jobId}...`);
...
log.debug(`> Forecast for ${jobId} created`);
```

Reply:
https://github.com/elastic/kibana/pull/179321/commits/3df589231d5ea792d1d61d9e14d3bb541cf27f13